### ISSUE TITLE
Feature: Dashboard UI and backend integration

### ISSUE THREAD

**Issue by dgan181** [cryptopatron/web-app#19]

**Comment by anishnuni** [comment]
UI looks good. But one issue I saw: In the modal that appears on the sidebar the user clicks on their Metamask account, the pen doesn't do anything (fapen icon).

**Comment by dgan181** [comment]
Yeah. I wanted that there to be able to change your wallet address. I figured it would be something similar to the metamask login stuff so I left that be for now.
